Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -957,17 +959,19 @@ namespace OpenInfraPlatform {
 					// **************************************************************************************************************************
 					// Determine position
 					carve::math::Matrix conicPositionMatrix = placementConverter->convertIfcAxis2Placement(circle->Position);
+					carve::math::Matrix inverseConicPositionMatrix;
+					GeomUtils::computeInverse(conicPositionMatrix, inverseConicPositionMatrix);

```

Comment:
- change the existing function so that it returns the inverse;
- plus add doxy comments to it;

Reply:
Resolved:
d9ce668